assets/i18n/en_US.json

```diff
@@ -31,6 +31,7 @@
         "installUpdate": "Continue to install the update?",
 
         "updateDialogTitle": "Update Manager",
+        "updatePatchesDialogTitle": "Update Patches",
```

Comment:
```suggestion
        "updatePatchesDialogTitle": "Update ReVanced Patches",
```

Reply:
Adding `ReVanced` before Patches will make the UI inconsistent with line 33.

Reply:
"ReVanced Patches" is the "name". You can either refer to the patches as "the patches" or "ReVanced Patches" but not "the Patches" or "Patches".